### ISSUE TITLE
[Help] Broken mail server after avorted update process

### ISSUE THREAD

**Issue by jpbaril** [mailcow/mailcow-dockerized#626]
Hi,

Two days ago I attempted to update my mailcow installation (See https://github.com/mailcow/mailcow-dockerized/issues/619). At the end of the update process the containers were not able to restart as the updated docker-compose file required Docker engine 17.06 which my CoreOS VM does not have. Fortunately I had not deleted the old container images and I had a backup of my previous docker-compose file. So I just put back the old docker-compose file and started it.

I thought it worked as the containers started healthy and I could connect to my mails by IMAP or SOGo.

But now I realize that I cannot receive nor can I send emails anymore.

Thanks for your help.

**Comment by mkuron** [comment]
You'll also need to revert back to the git revision you had before the upgrade. Otherwise the containers are incompatible with the config files.

**Comment by jpbaril** [comment]
How do I do that?
I searched but could not figure out what command to use exactly. I saw commands like "git rev-list" and "git checkout" but it's not clear to me as I don't really know git well.

My last successful mailcow server update was on September 9.

BTW, if it helps, here is a message I received in my gmail account after I tried to email my mailcow server:

`Reporting-MTA: dns; googlemail.com
Received-From-MTA: dns; xxx@gmail.com
Arrival-Date: Wed, 20 Sep 2017 16:27:58 -0700 (PDT)
X-Original-Message-ID: <1505950076.6014.0.camel@gmail.com>

Final-Recipient: rfc822; xxx@somemailcowserver.net
Action: delayed
Status: 4.3.0
Remote-MTA: dns; mail.somemailcowserver.net. (xxx.xxx.xxx.xxx, the server for the domain somemailcowserver.net.)
Diagnostic-Code: smtp; 451 4.3.0 <xxx@gmail.com>: Temporary lookup failure`

Thank you very much mkuron!

**Comment by jpbaril** [comment]
So I tried to solve it by myself. As previously said, I tried commands like "git rev-list", "git checkout" and "git reset --hard" but that did not work. I don't know git well so I'm basically trying random stuff without really knowing what I'm doing.

Any help would be really appreciated. Thanks.

**UPDATE:** It seems to work in the end. I assumed that the file dates would show that of September 9, but if I look at their content they reflect the state they had on September 9.
